src/Components/Samples/BlazorServerApp/wwwroot/css/open-iconic/README.md

```diff
@@ -1,14 +1,14 @@
-[Open Iconic v1.1.1](http://useiconic.com/open)
+[Open Iconic v1.1.1](https://ionic.io/ionicons)
 ===========
 
-### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](http://useiconic.com/open#icons)
+### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://ionic.io/ionicons)
```

Comment:
Should we still refer to `Iconic` given that it's not there?
Highlighted below?
Or should we just change this text to not refer to `Iconic` any more for comparison.
```suggestion
### Open Iconic is the open source sibling of *[IONIC](http://ionic.io)*. It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://ionic.io/ionicons)
```

Reply:
Unless these readmes were copied from the appropriate version of the framework.